debian_package_manager/internal/deb/snapshot.go

```diff
@@ -55,7 +55,20 @@ var (
 func latest(urltemplate string) (string, error) {
 	year, month, _ := time.Now().Date()
 	snapshotURL := fmt.Sprintf(urltemplate, year, month)
-	resp, err := rhttp.Get(snapshotURL)
+	resp, err := latestFromUrl(snapshotURL)
+	if err != nil {
```

Comment:
I think if we're going to do this. We should probably handle the new year case as well. If (month == 1) or whatever